### ISSUE TITLE
Safari - Palme

### ISSUE THREAD

**Issue by ghost** [dfringeli/ktf2021#28]
Auf Safari wird die Palme gar nicht angezeigt links oben:

![image](https://user-images.githubusercontent.com/46542480/77455734-8ba02000-6dfa-11ea-846b-61890c3631cb.png)

**Comment by dfringeli** [comment]
Ja, weil der Trick mit svg ausblenden und path einblenden in Safari nicht geht.
Wenn ich das svg einblende, dann über deckt die Palme die Website. Siehe #14 

Wie verbleiben wir?
Soll ich es noch ein zweites Mal anschauen..?

**Comment by angie-graphics** [comment]
es muss bestimmt irgend einen anderen weg geben.. wäre super, wenn du bei gelegenheit nochmals prüfen könntest, ob du das hin kriegst. ohne palem sieht die webseite ganz anders aus. müsste schon überall gleich aussehen wenn möglich. sorry 

du schaffst das schon.. im web brauchts eben manchmal ein paar hacks für gewisse browser, aber es gibt meistens eine lösung. ;)

**Comment by dfringeli** [comment]
Ja ich kann es nochmal anschauen, aber ich werde keine etlichen Stunden mehr investieren - Es waren jetzt schon zu viele.
